app/src/main/java/com/edulectronics/tinycircuit/Views/CircuitActivity.java

```diff
@@ -83,9 +83,9 @@ public boolean onTouch(View v, MotionEvent event) {
         circuit.setAdapter(new CircuitAdapter(this));
     }
 
-    private void getController() {
-        Intent intent = getIntent();
+    private void getControllers() {
         circuitController = CircuitController.getInstance();
+        wireController = new WireController((DrawView) findViewById(R.id.draw_view));
```

Comment:
Is het een probleem als er meerdere wirecontrollers in de applicatie rondzweven? Zo ja dan geen new WireController doen maar oplossen met singleton of dependency injection